arrows/gdal/image_io.h

```diff
@@ -25,23 +23,23 @@ class KWIVER_ALGO_GDAL_EXPORT image_io
 public:
   // No configuration for this class yet
   /// \cond DoxygenSuppress
-  virtual void set_configuration(vital::config_block_sptr /*config*/) { }
-  virtual bool check_configuration(vital::config_block_sptr /*config*/) const { return true; }
+  virtual void set_configuration(vital::config_block_sptr //config
+) { }
+  virtual bool check_configuration(vital::config_block_sptr //config
+) const { return true; }
```

Comment:
This is something which should be looked at more closely

Reply:
Thanks for catching this.  I've reverted to the original form. Though I'm not thrilled with that version, it wasn't the intent to catch things like this.